release notes/v0.49.0.md

```diff
@@ -0,0 +1,202 @@
+k6 `v0.49.0` is here 🎉! This release includes:
+
+- a built-in web-dashboard displaying test results in real-time
+- add `clear` functionality to the `locator` classes within the browser module
+- gRPC experimental module being merged back
+- being able to get the selection from an element in `k6/html`
+- collect internal modules and outputs used by a script
+- getting `k6/experimental/timers` ready to be stabilized
+- a bunch of internal changes, most of which are fixing lint errors, but also making MacOS test work :tada: 
+
+
+## Breaking changes
+
+
+- [#3494](https://github.com/grafana/k6/pull/3494) stops pushing `loadimpact/k6` docker image. If you still use it please migrate to the  `grafana/k6`
```

Comment:
```suggestion
- [#3494](https://github.com/grafana/k6/pull/3494) stops pushing `loadimpact/k6` docker image. If you still use it please migrate to the [grafana/k6](https://hub.docker.com/r/grafana/k6) image.
```

Reply:
Maybe also use `stops updating` instead of `stops pushing`?